celery_utils/migrations/0002_djcelery_cleanup.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.core import management
+from django.db import migrations, models
+
+
+def forwards(apps, schema_editor):
+    """
+    A dirty, dirty hack to clean up the unused (for us) djcelery tables.
+    """
+    if not Migration.DIRTY_HACK_CONSTANT:
+        Migration.DIRTY_HACK_CONSTANT = True
+        management.call_command('migrate', 'djcelery', '--fake-initial')
```

Comment:
It may be worth adding a check here to ensure the tables are actually empty? If they aren't, we'd probably rather throw an exception than drop data.